### DISCUSSION TITLE
Subscribe Mutations EVENTS : dev only?

### DISCUSSION THREAD

**Discussion by breizhwave** [vuejs/pinia#2196] [Help and Questions]
Hi, I am aware that mutation events are dev only, as a matter of fact they do not work under SSG rendering, but I really need that in order to alter global state depending on UI interactions on form that are modeled on the state. Any workaround ? 
 
thanks

https://pinia.vuejs.org/api/interfaces/pinia.SubscriptionCallbackMutationPatchFunction.html#events

**Comment by inane** [comment]
same problem, do you have any new about workaround?
